src/rockstor/settings.py

```diff
@@ -470,3 +472,16 @@
 # Note that the following will capture the build os version.
 # For live updates (running system) we call distro.version() directly in code.
 OS_DISTRO_VERSION = distro.version()  # 3, 15.0 ,20181107
+
+# Django Debug Toolbar-related settings and confrguration
+INTERNAL_IPS = [
+    "127.0.0.1", # It seems requests always come from this when using DRF
+]
+
+# https://django-debug-toolbar.readthedocs.io/en/stable/configuration.html
+DEBUG_TOOLBAR_CONFIG = {
+    # Update to the latest AJAX request
+    # Without this, we can only catch the initial request
+    # which is not helpful in most of our cases
+    "UPDATE_ON_FETCH": True
+}
```

Comment:
```suggestion
DEBUG_TOOLBAR_CONFIG = {
    # Update to the latest AJAX request
    # Without this, we can only catch the initial request
    # which is not helpful in most of our cases
    "UPDATE_ON_FETCH": True
    # If using htmx, to retain toolbar handle
    # through page renders add this
    "ROOT_TAG_EXTRA_ATTRS": "hx-preserve"    
}
```

If the project is contemplating to use htmx (#2735 and #2821) then this would also be required (among other things of course to enable htmx overall), according to:
https://django-debug-toolbar.readthedocs.io/en/stable/tips.html#working-with-htmx-and-turbo